RELEASING.md

```diff
@@ -16,17 +16,15 @@
 1. Perform [manual testing][3] that needs the tagging (source maps).
 1. Publish:
     - Sonatype, bintray (`./script/publish.sh`)
-    - CLI to website (simple `scp`)
+    - [Publish the CLI][4]
```

Comment:
Won't they follow independent release cycles?

Reply:
Yes and no. We still need to publish a new `scalajs-cli` artifact with version number, say, 1.0.2, that depends on `scalajs-tools` 1.0.2, for easy consumption.

At least that's how I see things in the early days of 1.x. We can relax this coupling later.